framework.go

```diff
@@ -229,3 +310,35 @@ func (f *Framework) DeleteOrg(ctx context.Context, org *organization.Org) error
 
 	return nil
 }
+
+func (f *Framework) getAppAndValues(name, namespace string) (*applicationv1alpha1.App, *corev1.ConfigMap, error) {
```

Comment:
This could actually be a useful function when writing tests (such as to check the state of existing installs).

Might be worth exporting this function.

Reply:
Yeah I can see it being useful. Done